src/Singularizer.php

```diff
@@ -0,0 +1,29 @@
+<?php
+declare(strict_types=1);
+
+namespace Pokemon;
+
+use Doctrine\Common\Inflector\Inflector;
+use Doctrine\Inflector\InflectorFactory;
+
+class Singularizer
+{
+    public static function singularize(string $word): string
+    {
+        if (\class_exists(Inflector::class)) {
```

Comment:
Can we drop the backslash `\` here?

Reply:
Shall I drop that FQFN everywhere? I am quite used to it 😁 or shall we use use for it?

Reply:
I just see these two instances of `\class_exists`, did I miss any others? I think for consistency, I'd drop them.